src/MagnumPlugins/UfbxImporter/UfbxImporter.conf

```diff
@@ -0,0 +1,76 @@
+depends=AnyImageImporter
+provides=FbxImporter
+provides=ObjImporter
+
+# [configuration_]
+[configuration]
+
+# Generate normals if missing.
+generateMissingNormals=false
+
+# Fail if the file is slightly malformed.
+strict=false
+
+# Disable any exporter-specific quirks in the importer.
+disableQuirks=false
+
+# Attempt to load further files referenced by this file.
+# On .obj files this would include .mtl files.
+loadExternalFiles=true
+
+# Ignore all geometry (vertices, indices) during loading.
+ignoreGeometry=false
+
+# Ignore all animation data (curves) during loading.
+ignoreAnimation=false
+
+# Ignore embedded data (such as images) during loading.
+ignoreEmbedded=false
+
+# Ignore all the actual content in the file, only loading the scene information
+# such as object relationships and names.
+ignoreAllContent=false
+
+# Maximum amount of temporary memory in bytes to use, negative for unlimited.
+maxTemporaryMemory=-1
+
+# Maximum amount of result memory in bytes to use, negative for unlimited.
+maxResultMemory=-1
+
+# Preserve the virtual root node of the scene. May be required for unit/axis
+# conversions to work properly.
+preserveRootNode=false
+
+# Create intermediate nodes to handle FBX "geometric transforms" that offset
+# the meshes in the scene graph without affecting children.
+geometricTransformNodes=true
+
+# Generate indexed geometry instead of returning flat raw vertex data.
```

Comment:
```suggestion
# Deduplicate raw vertex data to an indexed mesh.
```

In my opinion this better implies that it's an additional step that may take some extra time.